caikit/runtime/service_generation/primitives.py

```diff
@@ -0,0 +1,117 @@
+"""
```

Comment:
I think this file was deleted (or renamed).... looks like you should check out the latest `caikit/caikit` off main before re-running the script again